src/v2/api/controllers/ScheduleController.ts

```diff
@@ -180,4 +188,46 @@ export class ScheduleController {
     const result = await this.scheduleService.createGroupEvent(body);
     return this.scheduleMapper.getEvent(result.event, result.discipline);
   }
+
+  @Access('groups.$groupId.events.get')
+  @Get('/groups/:groupId/events')
+  @ApiOkResponse({
+    type: EventsResponse,
+  })
+  @ApiBadRequestResponse({
+    description: `\n
+    InvalidEntityIdException:
+      Group with such id is not found`,
+  })
+  @ApiUnauthorizedResponse({
+    description: `\n
+    UnauthorizedException:
+      Unauthorized`,
+  })
+  @ApiForbiddenResponse({
+    description: `\n
+    NoPermissionException:
+      You do not have permission to perform this action`,
+  })
+  @ApiQuery({
+    type: Boolean,
+    name: 'showOwnSelective',
+  })
+  @ApiQuery({
+    type: Number,
+    name: 'week',
+    required: false,
+  })
+  async getGroupEvents (
+      @Request() req,
+      @Param('groupId', GroupByIdPipe) groupId: string,
+      @Query('showOwnSelective', ConvertToBooleanPipe) showOwnSelective,
+      @Query('week') week: number,
+  ) {
+    const result = await this.scheduleService.getGroupEvents(req.user.id, groupId, showOwnSelective, week);
+    return {
+      events: this.scheduleMapper.getEvents(result.events),
```

Comment:
describe returned value disciplineType correctly. Follow this example 
![image](https://github.com/fictadvisor/fictadvisor-api/assets/113940774/3c586c96-7fd5-442c-a170-de011e3d9cb2)